includes/widgets/playing-cards.php

```diff
@@ -0,0 +1,290 @@
+				'selectors' => [
+					'{{WRAPPER}} .elementor-playing-cards-wrapper .elementor-playing-card' => 'padding: {{TOP}}{{UNIT}} {{RIGHT}}{{UNIT}} {{BOTTOM}}{{UNIT}} {{LEFT}}{{UNIT}};',
+				],
+			]
+		);
+
+		$this->add_control(
+			'playing_card_gap',
+			[
+				'label' => esc_html__( 'Gap', 'elementor' ),
+				'type' => Controls_Manager::SLIDER,
+				'size_units' => [ 'px', '%', 'em', 'rem', 'vw', 'custom' ],
+				'selectors' => [
+					'{{WRAPPER}} .elementor-playing-cards-wrapper' => 'gap: {{SIZE}}{{UNIT}};',
+				],
+			]
+		);
+
+		$this->end_controls_section();
+
+	}
+	/**
+	 * Render HTML widget output on the frontend.
+	 *
+	 * Written in PHP and used to generate the final HTML.
+	 *
+	 * @since 1.0.0
+	 * @access protected
+	 */
+	protected function render() {
+		$cards = $this->get_settings_for_display( 'cards' );
+		?>
+		<div  class="elementor-playing-cards-wrapper" >
+			<?php
+			foreach ( $cards as $index => $item ) :
+				$card_count = $index + 1;
+				$card_color_class = '♠' === $item['playing_card_type'] || '♣' === $item['playing_card_type'] ? 'elementor-playing-card-black' : 'elementor-playing-card-red';
+				$playing_card_number_settings_key = $this->get_repeater_setting_key( 'playing_card_number', 'cards', $index );
+				$playing_card_type_setting_key = $this->get_repeater_setting_key( 'playing_card_type', 'cards', $card_count );
+				$playing_card_wrapper_setting_key = $this->get_repeater_setting_key( 'playing-card-wrapper', 'cards', $card_count );
+
+				$this->add_render_attribute( $playing_card_number_settings_key, [
+					'class' => [ 'elementor-playing-card-value', $card_color_class ],
+					'data-playing-card-number' => $item['playing_card_number'],
+					'tabindex' => '0',
+				]);
+
+				$this->add_render_attribute( $playing_card_type_setting_key, [
+					'class' => [ 'elementor-playing-card-value', $card_color_class ],
+					'data-playing-card-type' => $item['playing_card_type'],
+				]);
+
+				$this->add_render_attribute( $playing_card_wrapper_setting_key, [
+					'data-card-number' => $card_count,
+				]);
+				?>
+
+				<div class="elementor-playing-card" <?php $this->print_render_attribute_string( $playing_card_wrapper_setting_key ); ?> >
+					<div <?php $this->print_render_attribute_string( $playing_card_number_settings_key ); ?>>
+					<?php
+						echo esc_html( $item['playing_card_number'] . $item['playing_card_type'] );
+					?>
+					</div>
+				</div>
+			<?php endforeach; ?>
+		</div>
+		<?php
+	}
+
+	/**
+	 * Render HTML widget output in the editor.
+	 *
+	 * Written as a Backbone JavaScript template and used to generate the live preview.
+	 *
+	 * @since 2.9.0
+	 * @access protected
+	 */
+	protected function content_template() {
+		?>
+		<# if ( settings.cards )
+			var elementUid = view.getIDInt().toString().substr( 0, 3 ); #>
+
+		<div class="elementor-playing-cards-wrapper">
+			<# _.each( settings.cards, function( item, index ) {
+			var cardCount = index + 1,
```

Comment:
Use `const` or `let`. See e.g. https://javascript.plainenglish.io/es6-variable-definition-feature-using-let-and-const-14eb143f799e

Reply:
I did, but it was screaming at me not to do it  for some reason